freshmaker/pyxis.py

```diff
@@ -339,3 +339,32 @@ def image_is_tagged_auto_rebuild(self, nvr):
                 if set(tag['name'] for tag in repo['tags']) & set(auto_rebuild_tags):
                     return True
         return False
+
+    def remove_hotfix_images(self, bundle_nvrs):
```

Comment:
Please add an API like `is_hotfix_image(self, image_nvr)` in pyxis, then call the new API in some way like:
```
bundle_nvrs = [nvr in bundle_nvrs if not self._pyxis.is_hotfix_image(nvr)]
```

Reply:
ahh gotcha I see, calling a method to return a bool and creating a list comprehension off it is a much better way. I'll change my code accordingly!